src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
I guess ExternalKind is ok, although we might need to add an InternalKind eventually.